src/Components/data/TileContainer.js

```diff
@@ -18,27 +22,44 @@ const TileContainer = () => {
   }, [activeStep])
 
   const handleKeyDown = ({ key, code }) => {
-    if (key === 'ArrowUp') {
-      setActiveStep(prevActiveStep => prevActiveStep - 15)
-
-    } else if (key === 'ArrowRight') {
-      setActiveStep(prevActiveStep => prevActiveStep + 1)
-
-    } else if (key === 'ArrowDown') {
-      setActiveStep(prevActiveStep => prevActiveStep + 15)
-
-    } else if (key === 'ArrowLeft') {
-      setActiveStep(prevActiveStep => prevActiveStep - 1)
-
+    console.log(tilePosition);
```

Comment:
All of the state in TileContainer should be able to be moved out with Redux.  I also think we might be able to move our event listener and movement logic to another component or file, making it easier to manage Tiles being generated.